addon/components/base/bs-tab/pane.js

```diff
@@ -20,7 +20,6 @@ export default Component.extend(ComponentChild, TransitionSupport, {
   layout,
   classNameBindings: ['active', 'usesTransition:fade'],
   classNames: ['tab-pane'],
-  ariaRole: 'tabpanel',
```

Comment:
I am just a bit puzzled: didn't you want to remove the `role="presentation"` from a `nav.item`? This is something different here!?

Reply:
:100: Seems like I was a little bit tired when I prepared this PR. :confused: